### PR TITLE
Include additional test references for PvModel 

### LINKED ISSUE #590: Include additional test references for PvModel
At investigation on #574 additional references for angle calculations done in PvModel have been found. These could be added to enrich the tests e.g. for other locations (Africa, Southern hemisphere, ...) and generally improve the test basis for this model.

### PR DESCRIPTION
**Pull request by danielfeismann**
resolves #590

### CODE REVIEW

**Comment by danielfeismann** [comment]
!test

**Review by jo-bao — CHANGES_REQUESTED** [review]
It's day 264 => Maybe that changes the jSol?

File: src/test/groovy/edu/ie3/simona/model/participant/PvModelTest.groovy [hunk 1]
```diff
@@ -276,50 +307,78 @@ class PvModelTest extends Specification {
...
+    51.516667d    | +18.4557514d    | -15.00225713d  | 90d        | +0d        || 58.65287310017624d                     // Iqbal
+    35.0d         | +23.2320597d    | +30.00053311d  | 45d        | 10d        || 39.62841449023577d                      // Kalogirou - Solar Energy Engineering Example 2.7  ISBN 978-0-12-374501-9; DOI https://doi.org/10.1016/B978-0-12-374501-9.X0001-5
+    35.0d         | +23.2320597d    | +30.00053311d  | 45d        | 90d        || 18.946300807438607d                     // Kalogirou - Solar Energy Engineering Example 2.7 changed to 90° panel azimuth to WEST
+    35.0d         | +23.2320597d    | +74.648850625d | 45d        | 90d        || 21.95480347380729d                     // Kalogirou - Solar Energy Engineering Example 2.7  90° panel azimuth to WEST at 17:00
+    35.0d         | +23.2320597d    | +74.648850625d | 45d        | -90d       || 109.00780288303966d                     // Kalogirou - Solar Energy Engineering Example 2.7  90° panel azimuth to EAST at 17:00
+    27.96d        | -17.51d         | -11.1d         | 30d        | +10d       || 22.384603601536398d                     // Goswami Priciples of Solar Engineering Example 2.7a
```
Comment (jo-bao) [inline src/test/groovy/edu/ie3/simona/model/participant/PvModelTest.groovy:344]: In Goswami that's called panel azimuth angle -> Maybe it's something different?

**Review by jo-bao — CHANGES_REQUESTED** [review]
Looks good! There are some minor points.

File: src/test/groovy/edu/ie3/simona/model/participant/PvModelTest.groovy [hunk 2]
```diff
@@ -148,6 +146,10 @@ class PvModelTest extends Specification {
     '2019-01-05T05:15:00+01:00[Europe/Berlin]' || 0.06885682528415985d
     '2016-10-31T12:15:00+01:00[Europe/Berlin]' || 5.23311872159614873d // leap year => day = 305
     '2017-10-31T12:15:00+01:00[Europe/Berlin]' || 5.21590451527510877d // regular year => day = 304
+    '2011-06-21T13:00:00+02:00[Europe/Berlin]' || 2.9436292808978335d // regular year => day = 172
+    '2011-04-05T16:00:00+02:00[Europe/Berlin]' || 1.6181353941777565d // regular year => day = 95
+    '2011-09-21T00:00:00+02:00[Europe/Berlin]' || 4.5273362624335105d // regular year => day = 254
```
[the remark this answers is not in the document]
Comment (danielfeismann) [reply src/test/groovy/edu/ie3/simona/model/participant/PvModelTest.groovy:151]: True :)

File: src/test/groovy/edu/ie3/simona/model/participant/PvModelTest.groovy (hunk 1 above)
Comment (danielfeismann) [reply src/test/groovy/edu/ie3/simona/model/participant/PvModelTest.groovy:344]: Which exactly do you mean? Panel azimuth angle @ Goswami is alpha_w which represents the rotation angle between the east-west axis and the panel (Figure 2.21). That corresponds to our defintion. A panel with alpha_w = +10° should be turned slightly to the east from the south.

Comment (jo-bao) [reply src/test/groovy/edu/ie3/simona/model/participant/PvModelTest.groovy:344]: Oh, I think I have mixed up the names of the angles. Yours should be right.

**Review by jo-bao — DISMISSED** [review]
I think, now everything is fine :)

**Review by jo-bao — APPROVED** [review]
Top :)

**Pull request merged**